src/CMakeSettings.json

```diff
@@ -1,180 +1,223 @@
+                    "type": "BOOL"
+                }
+            ]
+        },
+        {
+            "name": "x64-Release",
+            "generator": "Visual Studio 17 2022 Win64",
+            "configurationType": "Release",
+            "buildRoot": "${projectDir}\\out\\build\\${name}",
+            "installRoot": "${projectDir}\\out\\install\\${name}",
+            "cmakeCommandArgs": "",
+            "buildCommandArgs": "",
+            "ctestCommandArgs": "",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "variables": [
+                {
+                    "name": "DHARTAPI_Config",
+                    "value": "All",
+                    "type": "STRING"
+                },
+                {
+                    "name": "DHARTAPI_EnableTests",
+                    "value": "False",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "CMAKE_BUILD_TYPE",
+                    "value": "Release",
+                    "type": "STRING"
+                },
+                {
+                    "name": "DHARTAPI_EnableCSharp",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "DHARTAPI_EnablePython",
+                    "value": "true",
+                    "type": "BOOL"
+                }
+            ]
+        },
+        {
+            "name": "x64-Release-Python",
+            "generator": "Visual Studio 17 2022 Win64",
+            "configurationType": "Release",
+            "buildRoot": "${projectDir}\\out\\build\\${name}",
+            "installRoot": "${projectDir}\\..\\build\\Python",
+            "cmakeCommandArgs": "",
+            "buildCommandArgs": "",
+            "ctestCommandArgs": "",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "variables": [
+                {
+                    "name": "DHARTAPI_Config",
+                    "value": "All",
+                    "type": "STRING"
+                },
+                {
+                    "name": "DHARTAPI_EnableTests",
+                    "value": "False",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "CMAKE_BUILD_TYPE",
+                    "value": "Release",
+                    "type": "STRING"
+                },
+                {
+                    "name": "DHARTAPI_EnablePython",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "DHARTAPI_EnableCSharp",
+                    "value": "False",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "INSTALL_GTEST",
+                    "value": "False",
+                    "type": "BOOL"
+                }
+            ]
+        },
+        {
+            "name": "x64-Release-CSharp",
```

Comment:
looks like this config is the same as the previous one.   Can you rebase the change or undo a whitespace change?